spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1,5 +1,8925 @@
+driving force behind the expression language introduced in JSTL, to
+James Strachan for leading the group in our understanding of XML for
+page authors, and to Craig McClanahan for his help on servlet and J2EE
+platform related issues.
+
+This specification has drawn a lot of its design
+ideas from pioneers in the field of tag libraries. We are grateful to
+the Jakarta project at Apache, as well as other efforts in the industry,
+where projects led by Craig McClanahan (Struts), James Strachan (XTags),
+Morgan Delagrange (DBTags), Tim Dawson (I18N), Glenn Nielsen (many
+utility taglibs), Scott Hasse (JPath), Dmitri Plotnikov (JXPath), Pasi
+Salminen (O&D Struts), have greatly influenced the design of the JSTL
+libraries.
+
+The RI team composed of Shawn Bayern (lead),
+Nathan Abramson, Justyna Horwat, and Jan Luehe has done a wonderful job
+at turning code faster than the specification could be written.
+
+Quality has been in the capable hands of Ryan
+Lubke, lead of the TCK team that also includes Lance Andersen. David
+Geary’s help in doing thorough reviews of the specification was also
+greatly appreciated.
+
+We are also grateful to the product team at Sun
+Microsystems who helped us sail efficiently through this specification:
+Jim Driscoll, Karen Schaffer, George Grigoryev, Stephanie Bodoff,
+Vanitha Venkatraman, Prasad Subramanian, and Xiaotan He.
+
+Finally, we'd like to thank the community at
+large for their ever increasing interest in this technology. We sure
+hope you’ll enjoy the JSP Standard Tag Library.
+
+== Comments
+
+We are interested in improving this
+specification and welcome your comments and suggestions. You can email
+your comments to us at:
+
+ jstl-dev@eclipse.org
+
+:sectnums:
 == Introduction
 
-Jakarta Standard Tag Library provides a specification document, API and TCK that extends the 
-Jakarta Server Pages specification by adding a tag library of JSP tags for common tasks, 
-such as XML data processing, conditional execution, database access, loops and internationalization. 
+Jakarta Standard Tag Library provides a specification document, API and TCK that extends the
+Jakarta Server Pages specification by adding a tag library of JSP tags for common tasks,
+such as XML data processing, conditional execution, database access, loops and internationalization.
+
+
+=== Goals
+
+The ultimate goal of JSTL is to help simplify
+JavaServer™ Pages (JSP™) page authors’ lives.
+
+A page author is someone who is responsible
+for the design of a web application’s presentation layer using JSP
+pages. Many page authors are not fluent in any programming language.
+
+One of the main difficulties a page author is
+faced with is the need to use a scripting language (the default being
+the Java programming language) to manipulate the dynamic data within a
+JSP page. Unfortunately, page authors often see scripting languages as
+complex and not very well adapted to their needs.
+
+JSTL offers the following capabilities:
```

Comment:
Replace JSTL reference

Reply:
Changed to: `The Jakarta Standard Tag Library offers the following capabilities:`